packages/components/scripts/vendorism.js

```diff
@@ -155,13 +155,13 @@ const config = {
       // add custom styles to the end of `${component}.styles.ts`
       (path, content) => {
         let newContent;
-        components.forEach((component) => {
+        [...components, 'form-control'].forEach((component) => {
```

Comment:
Where is this coming from? What are we doing with `form-control` here?

Reply:
the form-control is a component that is common to some components like the input and the textarea, since they are input components. It is a part of them.